features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/mssql.sql

```diff
@@ -23,5 +24,4 @@ CREATE TABLE IDN_REMOTE_FETCH_REVISIONS (
   PRIMARY KEY (ID),
   FOREIGN KEY (CONFIG_ID) REFERENCES IDN_REMOTE_FETCH_CONFIG(ID) ON DELETE CASCADE,
   CONSTRAINT UC_REVISIONS UNIQUE (CONFIG_ID,ITEM_NAME)
-);
-
+);
```

Comment:
Add a new line here. Please check other files as well.

Reply:
Thank you, This comment was fixed in
https://github.com/wso2-extensions/identity-fetch-remote/pull/40
https://github.com/wso2-extensions/identity-fetch-remote/pull/41
https://github.com/wso2-extensions/identity-fetch-remote/pull/42
https://github.com/wso2-extensions/identity-fetch-remote/pull/43
https://github.com/wso2-extensions/identity-fetch-remote/pull/44
https://github.com/wso2-extensions/identity-fetch-remote/pull/45.
I resolve this conversation.